src/ga4gh/vrs/dataproxy.py

```diff
@@ -144,32 +148,33 @@ def derive_refget_accession(self, ac: str):
                 refget_accession = aliases[0].split("ga4gh:")[-1]
 
         return refget_accession
-    
-    def is_valid_ref_seq(
-        self, sequence_id: str, start_pos: int, end_pos: int, ref: str
-    ) -> Tuple[bool, str]:
-        """Return wether or not the expected reference sequence matches the actual
-        reference sequence
 
+    def validate_ref_seq(
+        self, sequence_id: str, start_pos: int, end_pos: int, ref: str,
+        require_validation: bool = True
+    ) -> None:
+        """Determine wether or not the expected reference sequence matches the actual
+        reference sequence
```

Comment:
Just to make the top-level description a bit clearer, maybe something like

```suggestion
        """Determine whether or not the expected reference sequence matches the actual
        reference sequence. Nothing is returned, but invalid results are logged at level WARN
        by default. If ``require_validation`` is ``True``, then invalid data will cause a 
        ``DataProxyValidationError`` to be raised.
```

should add in a `:raises:` clause too (this will become very repetitive but it'll look nicer if we generate an API reference later)

Reply:
also you should manually re-type that, I have no idea how the line breaks will actually look